src/controller/createQuestionnaireController.js

```diff
@@ -29,10 +30,12 @@ const createQuestionnaireController = async (req, res) => {
 
       const newQuestionnaire = await createQuestionnaire(Questionnaire, body);
       if(newQuestionnaire.errors){
+        logger.info('Questionnaire not created due to incorrect data')
```

Comment:
It can be useful to log the incorrect data that produced the error (maybe at 'debug' level)